tests/dialects/test_tsql.py

```diff
@@ -5,6 +5,15 @@
 class TestTSQL(Validator):
     dialect = "tsql"
 
+    def test_create_table(self):
+        self.validate_identity("CREATE TABLE x (A INTEGER NOT NULL, B INTEGER NULL)")
```

Comment:
remove the identity test and just put it in the validate all test

delete this function and add it to the tsql tests